web/package.json

```diff
@@ -9,9 +9,15 @@
   },
   "scripts": {
     "start": "parcel",
-    "build": "parcel build"
+    "build": "parcel build",
+    "lint": "eslint src/",
+    "fix": "prettier --write src/ && eslint --fix src/"
```

Comment:
Not sure if it's just me but checking out this branch and trying to run this doesn't fix errors(which are fixable) for me.

Reply:
`npm run fix` isn't fixing things? The only errors I had to fix were documentation based.